src/www/ui_components/server-card.html

```diff
@@ -225,8 +234,8 @@
           }[state] || this.localize("connect-button-label")
         );
       },
-      _computeConnectButtonDisabled: function(state) {
-        return state === "CONNECTING" || state === "DISCONNECTING";
+      _computeConnectButtonDisabled: function(state, usesUnsupportedCipher) {
+        return !usesUnsupportedCipher || state === "CONNECTING" || state === "DISCONNECTING";
```

Comment:
We can revert this change, and add a check for a "disabled" state.

Reply:
Done